api/plan.go

```diff
@@ -230,6 +230,12 @@ type PlanPatch struct {
 	Type PlanType `jsonapi:"attr,type"`
 }
 
+type TrialPlanCreate struct {
```

Comment:
exported: exported type TrialPlanCreate should have comment or be unexported (revive)